### PR TITLE
Deprecate old AWS CD zone

### PR DESCRIPTION
**Pull request by freva**
@hmusum Changes to `ConfigServerRestExecutorImpl`
@jonmv 243eb0c

### CODE REVIEW

**Comment by hmusum** [comment]
👍

**Comment by jonmv** [comment]
You need to handle removal of the enum member, in ApplicationSerializer:390.

**Review by jonmv — CHANGES_REQUESTED** [review]
Read other comment :)

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/proxy/ConfigServerRestExecutorImpl.java [hunk 1]
```diff
@@ -89,7 +89,7 @@ public ProxyResponse handle(ProxyRequest proxyRequest) throws ProxyException {
                 .orElseGet(() -> new ArrayList<>(zoneRegistry.getConfigServerUris(zoneId)));
 
         StringBuilder errorBuilder = new StringBuilder();
-        if (queueFirstServerIfDown(allServers, proxyRequest)) {
+        if (allServers.size() > 1 && queueFirstServerIfDown(allServers, proxyRequest)) {
```
Comment (jonmv) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/proxy/ConfigServerRestExecutorImpl.java]: This was already checked in `queueFirstServerIfDown` ;0

Comment (freva) [reply controller-server/src/main/java/com/yahoo/vespa/hosted/controller/proxy/ConfigServerRestExecutorImpl.java]: Nice, removed.

**Comment by freva** [comment]
Changed serialization to ignore/skip `JobStatus` for non-existent `DeploymentJobs.JobType`s

**Review by jonmv — APPROVED** [review]

**Pull request merged**